app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -163,6 +261,67 @@ export default function Navigation() {
     </div>
   );
 
+  const menuItems = (
+    <div>
+      {menuDropDown(data).map(
+        ({
+          name,
+          notificationCount,
+          route,
+          target,
+          externalLink,
+          hasBottomDivider,
+        }) => {
+          const hasNotification =
+            notificationCount !== undefined && notificationCount > -1;
+
+          const linkContent = (
+            <MenuItem
+              classes={{
+                root: classNames(classes.menuItem, {
+                  [classes.menuItemMessage]: hasNotification,
+                  [classes.menuItemDivider]: hasBottomDivider,
+                }),
+              }}
+            >
+              {hasNotification ? (
+                <Badge
+                  color="primary"
+                  variant="dot"
+                  classes={{ badge: classes.badge }}
+                >
+                  <Typography noWrap>{name}</Typography>
+                </Badge>
+              ) : (
+                <Typography noWrap>{name}</Typography>
+              )}
+
+              {hasNotification ? (
+                <Typography
+                  noWrap
+                  variant="subtitle2"
+                  className={classes.notificationCount}
+                >
+                  {`${notificationCount} unseen`}
+                </Typography>
+              ) : null}
+            </MenuItem>
+          );
+
+          return externalLink ? (
+            <a href={route} key={name} target={target}>
+              {linkContent}
+            </a>
+          ) : (
+            <Link to={route} target={target} key={name}>
+              {linkContent}
+            </Link>
```

Comment:
I think the menu should be closed as soon as the user clicks on a link on it, just like how it works here on GitHub and on most other websites. Might have to add an onClick here to call `setMenuOpen(false)`